src/useWebReader.tsx

```diff
@@ -1,22 +1,21 @@
-import {
-  ActiveReader,
-  AnyConformsTo,
-  AxisNowEpubConformsTo,
-  LoadingReader,
-  WebpubManifest,
-  WebpubPdfConformsTo,
-} from './types';
 import React from 'react';
 import { fetchJson } from './utils/fetch';
 import HtmlReaderContent from './HtmlReader/HtmlReaderContent';
 import usePdfReader from './PdfReader';
 import useHtmlReader from './HtmlReader';
+import { ActiveReader, LoadingReader } from './types';
+import { WebpubManifest } from './WebpubManifestTypes/WebpubManifest';
```

Comment:
I would re-export `WebpubManifest` from `types/index.ts` so that we can import it from `./types` here instead of this more complicated import path